app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,11 @@
  *= require_tree .
  *= require_self
  */
+.tangerine {
```

Comment:
We should consider if we want to do this "the tailwind way" by modifying our `tailwind.config.js`. Perhaps this could be our `display` font, or perhaps we could introduce it as its own font family class. Here's the tailwind docs about [fonts](https://tailwindcss.com/docs/font-family).

This doesn't need to prevent this PR from merging, just a thought/suggestion, and we could always do it later.

Reply:
I have tried defining a tailwind font class in `tailwind.config.js` like this:
```js
module.exports = {
  theme: {
    extend: {
      fontFamily: {
        'tangerine': ['Tangerine', 'cursive']
      }
    }
  }
}
```

But it doesn't seem to be working. I apply a `font-tangerine` class to the `div` that I want styled with the **Tangerine Google WebFont**, but the font is not applied, and inspecting the browser console, I don't see any rules for a `.font-tangerine` class.

Does something need to be run after the `tailwind.config.js` file is changed, in order for the changes to be picked up? I tried `bin/rails assets:precompile` but it didn't help, I tried `npm run postcss:build` but I'm getting an error `missing script: postcss:build`.

Reply:
I wouldn't be using this font too much in the UI in any case, it's more for show on the home page for a nice first impact.